engine/crash/src/script_crash.cpp

```diff
@@ -170,13 +170,13 @@ namespace dmCrash
             lua_pushnumber(L, i+1);
 
             lua_newtable(L);
-            lua_pushstring(L, "name");
+            lua_pushliteral(L, "name");
```

Comment:
Thanks! I wasn't really aware of this macro before.

Since you asked questions about contributions offline, I'll elaborate some more here.
While the change is technically "idiomatic" Lua, the benefits are also quite small in terms of performance.
We sometimes leave things just because it's not a special gain, nor is it broken.
This PR lands on the plus side overall, as there is an actual official macro, and nothing else is needed.

Reply:
After your comments regarding build size for #9741, I checked size on this one too (`lua_pushliteral` is a macro that does `sizeof` like `dmHashLiteral`). It increased by ~500b.

Given this, you may want to revert this PR. The only instances that are hit often are the ones in the input code. But that's still only a few times each frame.